### ISSUE TITLE
Create MD5 after finalizing CRAM alignment

### ISSUE THREAD

**Issue by EddieLF** [populationgenomics/production-pipelines#408]
With the old pipeline, an md5 checksum file was created alongside each CRAM after it was aligned. Now, since the CRAM is uploaded to the bucket with a multipart transfer, the md5 checksum is not explicitly available on demand, nor is a .md5 file created.

It might be a good step to add in the pipeline to create the .md5 file after alignment and upload this file to the bucket alongside the CRAM itself, in case it is useful in future; e.g. for providing to collaborators so they can validate successful transfers.

**Comment by MattWellie** [comment]
@EddieLF  is this mitigated by first class files generating/recording a checksum in Metamist?

**Comment by EddieLF** [comment]
Yeah mostly. The first class files checksums use the crc32c hashes, but I've found people prefer the md5 checksums when downloading analysis files. But we can generate the md5s when they get moved to the release bucket when needed for a download. The crc32c checksum in Metamist FCF is fine validating them internally.
